formula/cita.rb

```diff
@@ -0,0 +1,42 @@
+# CITA Formula
```

Comment:
@kaikai1024 , I think is better to let this installation-related files in a separate repository. what do you think?

Reply:
yeah. seems that's better.
i will check it later